src/utils/roomManager.js

```diff
@@ -85,28 +86,7 @@ function generateLeaderboard(roomId) {
 
   return leaderboard;
 }
-function modifyPoints(roomId, pointModifiers) {
```

Comment:
This function was totally okay here and didnt need to be moved to another file. The whole point of keeping these functions in a separate file is to make the code more readable and keep the file, code structure clean